src/app/components/Text/index.jsx

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 import { string, bool, shape, node } from 'prop-types';
 import Markdown from 'markdown-to-jsx';
 import mediaQuery from '../../helpers/mediaQueries';
-import InlineLink from '../InlineLink';
+import InlineLink from '../../containers/InlineLink';
```

Comment:
This is more of a question about our application architecture - do we want components calling containers? 
I'd have thought the `Text` container should import the `InlineLink` container and pass it to the `Text` component.

Reply:
I see your point, and might come into a wider question on component design architecture that is out of scope of this PR. We already do something similar to this in other components.

Reply:
@jtart I have opened an issue to address this https://github.com/bbc/simorgh/issues/543

Reply:
@sareh I would agree with your point @sareh. However, wouldn't this approach be considered an anti-pattern since they separate two differing concerns?